src/Template/Wall/index.ctp

```diff
@@ -35,13 +35,9 @@
  * @link     https://tatoeba.org
  */
 
+echo $this->AssetCompress->script('wall.js', ['block' => 'scriptBottom']);
```

Comment:
No need to `echo` here because the script is stored in the `scriptBottom` block instead of being returned.